portal/component-tests/admin/user-edit.component-test.js

```diff
@@ -1,14 +1,14 @@
-const pageRenderer = require('../pageRenderer');
 const {
-  DATE: { LONDON_TIMEZONE },
-} = require('../../server/constants');
+  TIMEZONES: { LONDON },
+} = require('@ukef/dtfs2-common');
+const pageRenderer = require('../pageRenderer');
 
 const page = 'admin/user-edit.njk';
 const render = pageRenderer(page);
 
 describe(page, () => {
   const adminUser = {
-    timezone: LONDON_TIMEZONE,
+    timezone: LONDON,
```

Comment:
Some files now `TIMEZONES.LONDON`, others `LONDON`, can we be consistent please? I think either works

Reply:
updated